i18n/index.js

```diff
@@ -24,6 +24,9 @@ export default new NextI18Next({
 
     // cache the language in cookies and local storage
     caches: ['cookie', 'localStorage'],
+    cookieSameSite: 'none',
+    cookieSecure: true,
+    cookieExpirationDate: new Date(firstDay.getTime() + 7 * 24 * 60 * 60 * 1000),
```

Comment:
This does not work for me, but the following does:

```
    cookieMinutes: 7 * 24 * 60,
    cookieOptions: { path: '/', sameSite: 'none', secure: true },    
```

Also a secure cookie is only set if serving the website with SSL, so it won't working testing on localhost without SSL.

<img width="1608" alt="Bildschirmfoto 2021-03-02 um 18 41 54" src="https://user-images.githubusercontent.com/1532418/109690890-1bac2500-7b87-11eb-9993-eede86eba4f3.png">

Reply:
Unfortunately a secure cookies does not seem to solve the problem of showing errors accessing the locaStorage if embedding the website with an iframe from a different domain.

Reply:
We do have localstorage as the fallback if cookie doesn't work, but I guess in case of iframes we are not even getting that.